ambari-logsearch/ambari-logsearch-server/src/main/java/org/apache/ambari/logsearch/conf/LogSearchJerseyResourceConfig.java

```diff
@@ -18,19 +18,18 @@
  */
 package org.apache.ambari.logsearch.conf;
 
+import javax.ws.rs.ApplicationPath;
+
 import org.apache.ambari.logsearch.rest.ServiceLogsResource;
-import org.glassfish.jersey.jackson.JacksonFeature;
 import org.glassfish.jersey.server.ResourceConfig;
 import org.glassfish.jersey.servlet.ServletProperties;
 
-import javax.ws.rs.ApplicationPath;
-
 @ApplicationPath("/api/v1")
 public class LogSearchJerseyResourceConfig extends ResourceConfig {
 
   public LogSearchJerseyResourceConfig() {
     packages(ServiceLogsResource.class.getPackage().getName());
-    register(JacksonFeature.class);
+    register(com.fasterxml.jackson.jaxrs.json.JacksonJaxbJsonProvider.class);
```

Comment:
after that changed, every feature will work the same way? (lke if they . throwing an another exeption or the simple use cases)

Reply:
I checked the source of JacksonFeature and finally it registers JacksonJaxbJsonProvider too. But it also adds exception mappers for JsonMappingException and JsonParseException resulting a text/plain error response. This is what we do not want so we can provide our own exception mappers.